config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -857,6 +890,7 @@ postsubmits:
       - aramase
       - ritazh
   - name: secrets-store-csi-driver-e2e-gcp-postsubmit
+    cluster: eks-prow-build-cluster
```

Comment:
The gcp jobs are configured to use workload identity with the prow cluster. Moving it to the eks prow cluster would break this job?

cc @tam7t

Reply:
/hold

Reply:
@aramase @tam7t is that done via the service account?